crates/compiler/load_internal/src/module.rs

```diff
@@ -132,6 +136,10 @@ pub struct TypeCheckedModule<'a> {
 
 #[derive(Debug)]
 pub struct CheckedModule {
+    /// all aliases and their definitions. this has to include non-exposed aliases
+    /// because exposed aliases can depend on non-exposed ones)
+    pub aliases: MutMap<Symbol, (bool, Alias)>,
```

Comment:
What is the boolean? Also the comment is missing a parentheses somewhere and capitalization

Reply:
I'll remove it, along with the other aliases code